inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasUimaParser.java

```diff
@@ -203,107 +208,150 @@ else if (annotation instanceof Sentence) {
-                        // Add the UI annotation name to the index as an annotation.
+                        // Add the UI annotation name to the index as an
+                        // annotation.
                         // Replace spaces with underscore in the UI name.
-                        
+
                         MtasToken mtasAnnotation = new MtasTokenString(mtasId++,
                                 annotationUiName.replace(" ", "_") + MtasToken.DELIMITER,
                                 beginToken);
                         mtasAnnotation.setOffset(annotation.getBegin(), annotation.getEnd());
                         mtasAnnotation.addPositionRange(beginToken, endToken);
                         tokenCollection.add(mtasAnnotation);
 
-                        // Get features for this annotation, if it is indexed. First comes the
+                        // Get features for this annotation, if it is indexed.
+                        // First comes the
                         // internal feature name, then the UI feature name
-                        for (AnnotationFeature feature: layerFeatures.get(annotationName)) {
+                        for (AnnotationFeature feature : layerFeatures.get(annotationName)) {
                             String featureValue = "";
-                            // Test if the internal feature name is a primitive feature
+                            // Test if the internal feature name is a primitive
+                            // feature
                             if (WebAnnoCasUtil.isPrimitiveFeature(annotation, feature.getName())) {
-                                // Get the feature value using the internal name.
-                                // Cast to Object so that the proper valueOf signature is used by
-                                // the compiler, otherwise it will think that a String argument is
+                                // Get the feature value using the internal
+                                // name.
+                                // Cast to Object so that the proper valueOf
+                                // signature is used by
+                                // the compiler, otherwise it will think that a
+                                // String argument is
                                 // char[].
+
                                 featureValue = String.valueOf((Object) WebAnnoCasUtil
                                         .getFeature(annotation, feature.getName()));
-                                
-                                // Add the UI annotation.feature name to the index as an annotation.
-                                // Replace spaces with underscore in the UI name.
-                                
-                                MtasToken mtasAnnotationFeature = new MtasTokenString(mtasId++,
+
+                                String labelStr = null;
+                                if (feature.getUiName().equals("identifier")
```

Comment:
Please use braces after single expression ifs, else it is really confusing. also, you can move the conditions into one line.